doc/source/index.rst

```diff
@@ -16,6 +16,27 @@ Qibochem key features:
 * PySCF or Psi4 for calculation of 1- and 2-electron integrals
 * Unitary Coupled Cluster Ansatz
 
+
+How to Use the Documentation
+============================
+
+Welcome to the comprehensive documentation for ``Qibo``! This guide will help
```

Comment:
Heh, I think you mean ``Qibochem`` here.

Reply:
Yikes. That was copied from the template provided by Stefano. Updated.